packages/pigeon/lib/java_generator.dart

```diff
@@ -116,13 +117,13 @@ class JavaGenerator extends StructuredGenerator<JavaOptions> {
     indent.writeln('import io.flutter.plugin.common.StandardMessageCodec;');
     indent.writeln('import java.io.ByteArrayOutputStream;');
     indent.writeln('import java.nio.ByteBuffer;');
-    indent.writeln('import java.util.Arrays;');
     indent.writeln('import java.util.ArrayList;');
+    indent.writeln('import java.util.Arrays;');
     indent.writeln('import java.util.Collections;');
+    indent.writeln('import java.util.HashMap;');
     indent.writeln('import java.util.List;');
     indent.writeln('import java.util.Map;');
-    indent.writeln('import java.util.HashMap;');
```

Comment:
Optional: Consider replacing this with the approach I added to the C++ generator, which passes a list of imports to a helper that sorts them all then writes `import $foo` for each one, so we don't have to manually keep this sorted in the future.